### ISSUE TITLE
Standardize Git Workflow

### ISSUE THREAD

**Issue by krbundy** [nelsopet/lecospec#73]
Create and maintain a standardized Git workflow.  This will (likely) inlcude the following:

A master/main branch.  The main branch should always remain good-to-go.  No development should happen on main.

A 'dev' brand (where all development happens).  Dev is the only branch that gets pulled into master, and only once the branch is tested or validated as we decide is appropriate.  

Additional branches based on 'dev' for each enhancement, new feature, or bug.  These branches get merged into dev after testing.  All development should happen on these branches, and then get merged into dev.

**Comment by krbundy** [comment]
Appropriate branches have been created.
